readme.md

```diff
@@ -25,8 +25,11 @@ $ node-confluence-content-id-parser
 | --------------------- | -------------------------------------------------------------------------------------------------- | -------- | ---------------------- |
 | confluence-base-uri   | The base uri of your confluence instance: protocol, hostname and port                              | Yes      | -                      |
 | confluence-page-id    | The page id of your contents to be analyzed                                                        | Yes      | -                      |
-| confluence-username   | Your confluence username                                                                           | Yes      | -                      |
+| confluence-username   | Your confluence username, uses new Personal Access Token Authentication if none is given           | No       | -                      |
 | confluence-user-token | Your confluence user token                                                                         | Yes      | -                      |
 | output-directory      | The directory to which your content-id files will be writte                                        | No       | output                 |
 | recognition-pattern   | The pattern (regular expression), which will be used to determine which table contains content-ids | No       | `^Content-Id$`         |
 | recognition-flags     | The regular expression flags to be applied to the `recognition-pattern`                            | No       | `i` (case insensitive) |
+| identifier-column     | The index of the column containing the key to be used                           | No       | 0 |
+| starting-column     | The column to start indixing language codes from. This can be useful when you have a column containing comments or annotations                         | No       | 1 |
+| trim-content | Should all keys and translations values be automatically trimmed (e.g remove whitespaces at beginning and end) | No | true |
```

Comment:
@RTK Ich hatte Probleme beim Parsen von den Auto-generierten tabellen mit Yogi-Requirements, das .textContent vom HTMLElement hatte in den Tabellen-Zellen immer jede Menge whitespaces am Anfang und Ende.
Mit .trim() leicht zu bereinigen, hier war für mich aber die Frage ob das defaultmäßig eingeschaltet sein sollte. Teste das vllt mal gegen

Reply:
Finde ich ok, da Whitespaces eigentlich nichts ist, was explizit gepflegt wird.